assets/js/theme/custom/order-product-data.js

```diff
@@ -0,0 +1,55 @@
+/* @TODO: Need to figure out a way to hide API TOKEN on the front. */
+export default function getOrderProductData(apiToken) {
+    const selectors = {
+        item: '.js-card-order',
+        url: '.js-dynamic-url',
+        wishlist: '.js-wishlist-add',
+    }
+
+    const items = Array.from(document.querySelectorAll(selectors.item))
+    items.forEach((item) => {
```

Comment:
Maybe check if items exist first as an extra precaution, unless you are sure they will exist 100% of the times